src/class/eventManagement/incomer.class.ts

```diff
@@ -97,6 +98,8 @@ function isUnresolvedEvent(value: EventCallbackResponse): value is EventCallback
 export type IncomerOptions<T extends GenericEvent = GenericEvent> = {
   /* Service name */
   name: string;
+  redis: Types.DatabaseConnection<RedisAdapter>;
+  subscriber: Types.DatabaseConnection<RedisAdapter>;
```

Comment:
```suggestion
  publisher: Types.DatabaseConnection<RedisAdapter>;
  subscriber: Types.DatabaseConnection<RedisAdapter>;
```

Je me demande si c'est pas plus précis comme ça avec explicitement le pub et le sub, wdyt? (et si on fait ça pareil pour les autres classes)